edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -31,45 +31,57 @@ <h3>
       {% endif %}
       <button id="software_download_link" class="exam-action-button btn-pl-primary" data-action="click_download_software">{% trans "Start System Check" %}</button>
     {% else %}
-    <p>
-      {% blocktrans %}
-        1. Copy this unique exam code. You will be prompted to paste this code later before you start the exam.
-      {% endblocktrans %}
-    </p>
+      <h4>
+        {% blocktrans %}
+          Step 1
+        {% endblocktrans %}
+      </h4>
+      <p>
+        {% blocktrans %}
+          Copy this unique exam code. You will be prompted to paste this code later before you start the exam.
+        {% endblocktrans %}
+      </p>
     <p>
       <span class="proctored-exam-code">{{exam_code}}</span>
     </p>
     <p>
       {% blocktrans %}
-        Select the exam code, then copy it using Command+C (Mac) or Control+C (Windows).
+        Select the exam code, then copy it using Control + C (Windows) or Command + C (Mac).
```

Comment:
Highlighting this change for @stroilova to make sure she's aware this copy was changed.